.github/workflows/deploy.py

```diff
@@ -466,37 +465,6 @@ def run_uniswap_test(project_name):
         raise RuntimeError(f"Uniswap tests failed. Err: {out.stderr}")
 
 
-@cli.command(name="trigger_dapps_tests", help="Run dapps tests workflow")
-@click.option("--solana_ip", help="solana ip")
-@click.option("--proxy_ip", help="proxy ip")
-@click.option('--pr_url_for_report', default="", help="Url to send the report as comment for PR")
-@click.option('--token', help="github token")
-@click.option('--full_test_suite', help="set in true to run all dapps tests")
-def trigger_dapps_tests(solana_ip, proxy_ip, pr_url_for_report, token, full_test_suite):
-    github = GithubClient(token)
-
-    runs_before = github.get_dapps_runs_list()
-    runs_count_before = github.get_dapps_runs_count()
-    proxy_url = f"http://{proxy_ip}:9090/solana"
-    solana_url = f"http://{solana_ip}:8899/"
-    faucet_url = f"http://{proxy_ip}:3333/"
-
-    github.run_dapps_dispatches(proxy_url, solana_url, faucet_url, pr_url_for_report, full_test_suite)
-    wait_condition(lambda: github.get_dapps_runs_count() > runs_count_before, timeout_sec=180)
-
-    runs_after = github.get_dapps_runs_list()
```

Comment:
let's also delete all connected to old logic like "github.get_dapps_runs_list"

Reply:
Can you please specify one example of it, as it seems that the old logic was located in `github_api_client` file, which is deleted already.